dependencies/graknlabs/BUILD

```diff
@@ -13,4 +13,6 @@
 #
 # You should have received a copy of the GNU Affero General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
-#
\ No newline at end of file
+#
+
+exports_files(["graknlabs_console.BUILD"], visibility = ["//visibility:public"])
```

Comment:
Why do we need this exported file, @adammitchelldev ? It doesn't show up in this PR either.

Reply:
Ah, was for a previous method, good catch